src/linker/Linker/LinkContext.cs

```diff
@@ -235,6 +237,11 @@ public LinkContext (Pipeline pipeline, AssemblyResolver resolver, ReaderParamete
 			Optimizations = new CodeOptimizationsSettings (defaultOptimizations);
 		}
 
+		public void GenerateWarningSuppressions ()
```

Comment:
I would rename this so it doesn't sound like it is doing the generation here, and also prevent allocating a new writer in case it is called multiple times.